src/client/components/vis-measure-label/vis-measure-label.scss

```diff
@@ -21,3 +21,13 @@
   pointer-events: none;
   white-space: nowrap;
 }
+
+.measure-value {
```

Comment:
totals are affected, I prefer previous version

Reply:
Fixed in e3490a6